README.md

```diff
@@ -118,33 +111,45 @@ The repository consists of four parts:
 More details for each of these parts can be found here: [datasets](datasets/README.md), [models](model_interfaces/README.md), [runner](runner/README.md), [reports](reporting/README.md).
 
 
-## Benchmark 2 - 03/2024
+## Benchmark 3 - 04/2024
```

Comment:
The section "Configurations" seems outdated. Also, what do you think about placing the old configuration files under `configurations/older` or something like that?